### PR TITLE
Added Jest testing for reducers

### PR DESCRIPTION
**Pull request by kagehina919**
Towards #317.
Added more tests for reducers.
@wlach

### CODE REVIEW

File: frontend/test/reducers/reducers.spec.js [hunk 1]
```diff
@@ -13,4 +18,57 @@ describe('channelPlatformSummary reducer', () => {
...
+      channelPlatformSummary: {},
+      measures: { isFetching: true },
+    };
+    const receivedState = rootReducer(initialState, action);
+
+    expect(expectedState).toEqual(receivedState);
+  });
+
+  it('Recieves measure data', () => {
+    const initialState = {};
```
Comment (wlach) [inline frontend/test/reducers/reducers.spec.js]: Let's set a more realistic `initialState` here:

```js
const initialState = {
  channelPlatformSummary: {},
  measures: {
    isFetching: true
  }
}
```

File: frontend/test/reducers/reducers.spec.js [hunk 2]
```diff
@@ -13,4 +18,57 @@ describe('channelPlatformSummary reducer', () => {
 
     expect(expectedState).toEqual(receivedState);
   });
+
+  it('Receives channel platform summary data', () => {
```
Comment (wlach) [inline frontend/test/reducers/reducers.spec.js]: Could you add a test for `REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA` here, sort of parallel to what you did below with `REQUEST_MEASURE_DATA`?

**Comment by kagehina919** [comment]
@wlach requested changes made. Test for REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA already added to `reducers.spec.js` in the previous commit. Please review.

**Comment by wlach** [comment]
> @wlach requested changes made. Test for REQUEST_CHANNEL_PLATFORM_SUMMARY_DATA already added to reducers.spec.js in the previous commit. Please review.

@kagehina919 oh right! sorry I missed that

**Pull request merged**